modules/php/Models/Scoresheet.php

```diff
@@ -127,6 +128,18 @@ public function countScribblesInSection(string $section, ?int $type = null): int
     return $this->countScribbledSlots($this->getSectionSlots($section), $type);
   }
 
+  public function countAllUnscribbledSlots(): int
+  {
+    $allSlots = array_merge(...$this->getSections());
+    $unscribbledSlots = $this->getAllUnscribbled($allSlots);
+    return count($unscribbledSlots);
+  }
+
+  public function getSections(): array
+  {
+    return $this->increasingConstraints;
```

Comment:
Oh wait, forget about my previous comment but please use another terminology than "sections" because these refer to "slot sections" so it's a bit confusing

Reply:
Renamed